msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/MessageForumsForumManagerImpl.java

```diff
@@ -1527,6 +1527,7 @@ public String getAllowedGroupForRestrictedTopic(final Long topicId, final String
 				.getNamedQuery("findAllowedGroupInTopic")
 				.setLong("id", topicId)
 				.setString("permissionLevelName", permissionName)
+				.setMaxResults(1)
 				.uniqueResult();
```

Comment:
this should be a list() as more than one group/section should be returned.

Reply:
Done!

Reply:
@ern Is there any problem with the changes?

Reply:
Earle has a huge refactoring of msgcntr queued up ... with huge performance gains.

Also, I was never able to replicate the reported bug. I tried several times on my local trunk  .....

Reply:
@ottenhoff I uploaded a video with the problem. Have you seeen it?

Reply:
Yes, I watched several times and followed the steps several times.

Reply:
We have reproduced this issue in our Sakai 20 in production. We introduced a new ticket (SAK-44462) but @jesusmmp has advised that our new ticket in fact duplicated SAK-44292 Maybe our testing plan in SAK-44462 can help to reproduce the issue

Reply:
I was able to replicate today. @ern and I are looking at it

Reply:
yes looking